tests/integration/link/test_title_prop.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import dash
+from dash.dependencies import Input, Output
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+@pytest.mark.DCC768
+def test_title001_prop(dash_dcc):
```

Comment:
```suggestion
def test_liti001_prop(dash_dcc):
```
Per my comment on #776